SNSKY/donations.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- ticker: SNSKY
+    recipient:  
```

Comment:
Indent is wrong. Just remove 2 spaces to make it valid yaml.
You can use http://www.yamllint.com/ for validating yaml structure upfront.

Reply:
thanks for the hint. i have changed and checked on yamllimit and it should work now.